container/health.go

```diff
@@ -19,12 +19,7 @@ func (s *Health) String() string {
 		return types.Unhealthy
 	}
 
-	switch s.Status {
-	case types.Starting:
-		return "health: starting"
```

Comment:
@cpuguy83 this is user facing in the sense it's displayed in `docker ps` currently. That's currently the only use of it.